src/v/coding-style.md

```diff
@@ -6,12 +6,50 @@ The document covers style and convention not enforced by clang-format.
 
 Header files have the `.h` extension, source files use the `.cc` extension.
 
+Public header files should go in an `includes` directory within the module they belong to. 
```

Comment:
```suggestion
Public header files should go in an `include` directory within the module they belong to. 
```

`include` is overwhelmingly more common and is the de-facto standard in C and C++ AFAICT. Let's use that?

Reply:
Ah yes good catch - thanks, we're already using `include`.